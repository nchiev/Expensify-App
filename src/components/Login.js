import React from 'react';
import { NavLink } from "react-router-dom";

export default class Login extends React.Component {
    constructor(props) {
        super(props);
    }

    render() {
        return (
            <div>
                <form>
                    <button>Login</button>
                </form>
            </div>
        )
    }
}