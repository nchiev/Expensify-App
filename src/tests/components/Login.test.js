import React from 'react';
import { shallow } from 'enzyme';

import Login from '../../components/Login';

test('should render Login component', () => {
    const wrapper = shallow(<Login />);

    expect(wrapper).toMatchSnapshot();
});