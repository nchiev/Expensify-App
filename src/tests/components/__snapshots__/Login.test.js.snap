// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Login component 1`] = `
<div>
  <form>
    <button>
      Login
    </button>
  </form>
</div>
`;
